docs/how-to/h-enable-monitoring.md

```diff
@@ -0,0 +1,215 @@
+alertmanager:replicas                         alertmanager:replicas                            alertmanager_replica       peer     
+alertmanager:self-metrics-endpoint            prometheus:metrics-endpoint                      prometheus_scrape          regular  
+catalogue:catalogue                           alertmanager:catalogue                           catalogue                  regular  
+catalogue:catalogue                           grafana:catalogue                                catalogue                  regular  
+catalogue:catalogue                           prometheus:catalogue                             catalogue                  regular  
+grafana:grafana                               grafana:grafana                                  grafana_peers              peer     
+grafana:metrics-endpoint                      prometheus:metrics-endpoint                      prometheus_scrape          regular  
+loki:grafana-dashboard                        grafana:grafana-dashboard                        grafana_dashboard          regular  
+loki:grafana-source                           grafana:grafana-source                           grafana_datasource         regular  
+loki:metrics-endpoint                         prometheus:metrics-endpoint                      prometheus_scrape          regular  
+prometheus:grafana-dashboard                  grafana:grafana-dashboard                        grafana_dashboard          regular  
+prometheus:grafana-source                     grafana:grafana-source                           grafana_datasource         regular  
+prometheus:prometheus-peers                   prometheus:prometheus-peers                      prometheus_peers           peer     
+traefik:ingress                               alertmanager:ingress                             ingress                    regular  
+traefik:ingress                               catalogue:ingress                                ingress                    regular  
+traefik:ingress-per-unit                      loki:ingress                                     ingress_per_unit           regular  
+traefik:ingress-per-unit                      prometheus:ingress                               ingress_per_unit           regular  
+traefik:metrics-endpoint                      prometheus:metrics-endpoint                      prometheus_scrape          regular  
+traefik:traefik-route                         grafana:ingress                                  traefik_route              regular  
+
+```
+
+After the correct deployment of the Observability bundle, some other extra applications need to be deployed to enable the monitoring of the Charmed Spark solution.
+
+The first component is the Prometheus Pushgateway:
+```shell
+juju deploy prometheus-pushgateway-k8s --channel edge
+
+```
+
+followed by the followed by the `cos-configuration-k8s` and `prometheus-scrape-config-k8s`:
```

Comment:
Maybe, being an how-to, it would make a section "Configuring Observability Bundle", where we mark this optional. Something like:

```
# Configuring Observability Bundle

The observability bundle can be further customized using additional charms:

* `cos-configuration-k8s` in order to configure Grafana with custom dashboards that are stored in external repositories:

juju deploy cos-configuration-k8s \
  --config git_repo=<repository> \
  --config git_branch=<branch-name> \
  --config git_depth=1 \
  --config grafana_dashboards_path=<path>
juju relate cos-configuration-k8s grafana

We provide some default Spark dashboard [here](https://github.com/canonical/charmed-spark-rock/tree/dashboard/dashboards/prod/grafana)

```

This way it should sounds a bit more like an how-to, than a tutorial, that is a bit the impression here